conf/messages.en

```diff
@@ -377,7 +381,11 @@ removePlatformOperator.error.required = Select yes if you want to remove {0}
 
 platformOperatorRemoved.title = Platform operator has been successfully removed
 platformOperatorRemoved.heading = {0} (PO {1}) has been successfully removed
-platformOperatorRemoved.button = Back to view platform operators
+platformOperatorRemoved.button = Return to view platform operators
+platformOperatorRemoved.p1.1.two.emails = We have sent a confirmation email to {0} and {1}.
+platformOperatorRemoved.p1.1.one.email = We have sent a confirmation email to {0}.
+platformOperatorRemoved.emailNotSent.warning = You will not be emailed a confirmation of this.
+platformOperatorRemoved.print.link.text = Print this page
```

Comment:
use existing 'site.printThisPage' instead of creating a specific print link text for platformOperatorRemoved